b2b_ecommerce/views.py

```diff
@@ -1,6 +1,7 @@
 """Views for business to business ecommerce"""
 
 import logging
+from itertools import chain
```

Comment:
Having `csv_rows` and `instructions` separately will remove the need for chaining the iterables.